docs/Trident-API-Overview.md

```diff
@@ -139,6 +139,21 @@ Of course these operations can be chained, so a stream of uppercase words can be
 ```java
 mystream.flatMap(new Split()).map(new UpperCase())
 ```
+
```

Comment:
May be add a note above for the older implementation to call out that outputFields is same as inputFields.